contracts/utils/addresses.js

```diff
@@ -24,17 +24,21 @@ addresses.mainnet.COMP = "0xc00e94Cb662C3520282E6f5717214004A7f26888";
 addresses.mainnet.cDAI = "0x5d3a536e4d6dbd6114cc1ead35777bab948e3643";
 addresses.mainnet.cUSDC = "0x39aa39c021dfbae8fac545936693ac917d5e7563";
 addresses.mainnet.cUSDT = "0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9";
-
+// Curve
+addresses.mainnet.CRV = "0xd533a949740bb3306d119cc777fa900ba034cd52";
+addresses.mainnet.CRVMinter = "0xd061D61a4d941c39E5453435B6345Dc261C2fcE0";
+addresses.mainnet.ThreePool = "0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7";
+addresses.mainnet.ThreePoolToken = "0x6c3f90f043a72fa612cbac8115ee7e52bde6e490";
+addresses.mainnet.ThreePoolGauge = "0xbFcF63294aD7105dEa65aA58F8AE5BE2D9d0952A";
 // Open Oracle
-addresses.mainnet.openOracle = "0x922018674c12a7f0d394ebeef9b58f186cde13c1";
+addresses.mainnet.openOracle = "0x9b8eb8b3d6e2e0db36f41455185fef7049a35cae";
```

Comment:
I don't think this should be updated. The current feed is 0x922018674c12a7f0d394ebeef9b58f186cde13c1